ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -89,6 +89,13 @@ kmeansCommand
     k=integerLiteral
     ;
 
+adCommand
+    : AD
```

Comment:
We should support the other parameters. You can do it in next PR.

Reply:
Opened an issue to track it - https://github.com/opensearch-project/sql/issues/462
Will address this in the PR.